src/Weasel.Postgresql/Tables/IndexDefinition.cs

```diff
@@ -436,20 +452,47 @@ private static string canonicizeColumn(string expression)
             return CanonicizeCast(expression);
         }
 
-        private static string removeSortOrderFromExpression(string expression, out SortOrder order)
+        private static string removeSortOrderFromExpression(string expression, out SortOrder order, out NullsSortOrder? nullsSort)
         {
```

Comment:
Would the new C# pattern matching stuff help here maybe? Not a big thing.

Reply:
yeah, this nicely fit in the pattern matching. Fixed and addressed it.